packages/manager/modules/vps/src/dashboard/migrate/migrate-confirm/migrate-confirm.controller.js

```diff
@@ -0,0 +1,40 @@
+import get from 'lodash/get';
+import { MIGRATE_CONFIRMATION_INPUT_PATTERN } from './migrate-confirm.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($translate, VpsMigrationService) {
+    this.$translate = $translate;
+    this.MIGRATE_CONFIRMATION_INPUT_PATTERN = MIGRATE_CONFIRMATION_INPUT_PATTERN;
+    this.vpsMigrationService = VpsMigrationService;
+  }
+
+  cancel() {
+    this.goBackToMigrate();
+  }
```

Comment:
Why not directly use `$ctrl.goBackToMigrate` ?

Reply:
i added this with `click` tracking in mind